src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -200,6 +205,35 @@
     [react/view style/not-sent-icon
      [icons/icon :main-icons/warning {:color colors/red}]]]])
 
+(defn pin-author-name [pinned-by]
+  (let [user-contact @(re-frame/subscribe [:multiaccount/contact])
+        contact-names @(re-frame/subscribe [:contacts/contact-two-names-by-identity pinned-by])]
+    (if (= pinned-by (user-contact :public-key)) (i18n/label :t/You) (first contact-names))))
+
+(def pin-icon-width 9)
+
+(def pin-icon-height 15)
+
+(defn pinned-by-indicator [outgoing display-photo? pinned-by]
+  [react/view {:style (style/pin-indicator outgoing display-photo?)
+               :accessibility-label :pinned-by}
+   [react/view {:style {:flex-direction :row
+                        :align-items    :center}}
+    [icons/icon :main-icons/pin {:color            colors/gray
+                                 :height           pin-icon-height
+                                 :width            pin-icon-width
+                                 :background-color :red}]
+    [quo/text {:weight :regular
+               :size   :small
+               :color  :main
+               :style  (style/pinned-by-text)}
+     (i18n/label :t/pinned-by)]
+    [quo/text {:weight :medium
+               :size   :small
+               :color  :main
+               :style  (style/pin-author-text)}
+     (pin-author-name pinned-by)]]])
```

Comment:
why is it a function call here?

Reply:
It is a basic function which returns `You` text if public-key is the current account key, or the contact name otherwise.